packages/pyxis-react/src/components/Form/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,222 @@
+  },
+);
+
+const getResetClasses = (size: FieldSize): string => classNames(
+  'form-field__addon__reset',
+  {
+    'form-field__addon__reset--small': size === 'small',
+  },
+);
+
+export const Autocomplete: FC<AutocompleteProps> = forwardRef<HTMLInputElement, AutocompleteProps>(({
+  className,
+  disabled,
+  dropdownHeader,
+  errorMessage,
+  filteringFunction = 'startsWith',
+  hasError,
+  hint,
+  id,
+  isCaseSensitive = false,
+  items,
+  name,
+  noResultFoundAction,
+  noResultFoundText,
+  onValueChange,
+  onChangeFilteringValue,
+  size = 'medium',
+  suggestion,
+  value,
+  ...restProps
+}, ref: ForwardedRef<HTMLInputElement>) => {
+  const internalRef = useRef<HTMLInputElement>(null);
+  const combinedRef = useCombinedRefs(ref, internalRef);
+  const initialSelectedValue = getItemFromValue(items, value);
+  const initialFilteringValue = initialSelectedValue?.label || '';
+
+  const [selectedItem, setSelectedItem] = useState<Item | undefined>(initialSelectedValue);
+  const [filteringValue, setFilteringValue] = useState<string>(initialFilteringValue);
+  const [filteredItems, setFilteredItems] = useState<Item[] | undefined>(
+    filterItems(initialFilteringValue, filteringFunction, items, isCaseSensitive),
+  );
+  const [{ isOpen: isDropdownOpen, activeDescendant }, setDropdown, handleMouseEnter] = useDropdown(filteredItems);
+  const autocompleteId = id || name;
+  const dropdownProps = {
+    size, noResultFoundText, noResultFoundAction, suggestion, dropdownHeader,
+  };
+
+  useClickOutside(
+    [`#${getFieldId(autocompleteId)}`, `[for=${autocompleteId}]`],
+    () => setDropdown(initialDropdownConfig),
+    isDropdownOpen,
+  );
+
+  useEffect(
+    () => {
+      if (selectedItem) {
+        setFilteringValue(selectedItem.label);
+        if (onValueChange) onValueChange(selectedItem);
+      }
+    },
+    [selectedItem, onValueChange],
+  );
+
+  useEffect(
+    () => {
+      if (!items) setFilteredItems(undefined);
+      setFilteredItems(
+        filterItems(filteringValue, filteringFunction, items, isCaseSensitive),
+      );
+    },
+    [filteringFunction, isCaseSensitive, filteringValue, items],
+  );
+
+  const handleReset = () => {
+    setSelectedItem(undefined);
+    setFilteringValue('');
+    setDropdown(initialDropdownConfig);
+    if (onChangeFilteringValue) onChangeFilteringValue('');
+  };
+
+  const handleSelection = (selectedItemValue: string) => {
+    setSelectedItem(getItemFromValue(items, selectedItemValue));
+    setDropdown(initialDropdownConfig);
+  };
+
+  const handleChangeFilter = ({ target }: ChangeEvent<HTMLInputElement>) => {
+    if (selectedItem) setSelectedItem(undefined);
+    if (onChangeFilteringValue) onChangeFilteringValue(target.value);
+    setFilteringValue(target.value);
+  };
+
+  const handleClick = () => setDropdown(
+    ({ isOpen }) => ({
+      isOpen: !isOpen,
+      activeDescendant: filteredItems && filteredItems[0] ? filteredItems[0].value : undefined,
```

Comment:
This should do the trick:
```suggestion
      activeDescendant: filteredItems && filteredItems[0]?.value
```

Reply:
You can shorten it further to `activeDescendant: filteredItems?.[0]?.value`